### ISSUE TITLE
Why do we need RequestType.java and Status.java ?

### ISSUE THREAD

**Issue by hendrikebbers** [hiero-ledger/hiero-sdk-java#2040]
The two classes (identical in JS and other SDKs) look strange. The classes are wrappers around protobuf classes that created by hand but do not provide any additional information. Can we please check if the classes are really needed? I assume the classes (enums) created by protobuf are already good enough and we do not need the additonal handmade classes.

**Comment by 0xivanov** [comment]
Hi @hendrikebbers . It seems like `Status` could be removed, but I am not sure if there are not any users that are using the `Status` struct specifically for comparison.
`RequestType` would not be directly used, I think.

**Comment by hendrikebbers** [comment]
Yeah, but the question is the usage of RequestType can not be refactored to us the protobuf classes directly.

**Comment by 0xivanov** [comment]
We could use the protobuf classes directly, but I cannot guarantee that someone is not using our own implementation, since they are public classes.
We could `@Deprecate` them and use the protobufs. After some time we could remove the 2 deprecated classes.

**Comment by 0xivanov** [comment]
Since these APIs are public, we shall not remove or change them.
